bbtc.py

```diff
@@ -51,13 +48,14 @@ async def main():
 
     print('Finding device...')
```

Comment:
this is now printed even if we run straight to CLI prompt without scanning/connecting